docs/design.md

```diff
@@ -0,0 +1,41 @@
+# Design
+
+In the repository, GitHub Actions is used to manage the trestle workspace. This document describes the purpose and design of each workflow.
+
+## Workflows
+
+#### Create
+
+Associated Workflows
+- [create-new.yml](../.github/workflows/create-new.yml)
+
+The `create-new` workflow is triggered manually by going to the Action tab. This creates a new OSCAL profile in the trestle workspace. A new branch is created and a pull request is opened.
+
+#### AutoFix
```

Comment:
Would it ease understanding if this was titled "AutoSync" to align with the trestle-bot terminology?